litmus-portal/frontend/jest.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  preset: 'ts-jest/presets/js-with-ts',
+  testMatch: [
+    '<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}',
```

Comment:
we are using create react app...which already have jest config I think....is this config required then?
Try to remove this and check if everything works the same or not.